lib/node_modules/@stdlib/string/next-code-point-index/README.md

```diff
@@ -0,0 +1,205 @@
+limitations under the License.
+
+-->
+
+# nextCodePointIndex
+
+> Return the position of the next Unicode code point in a string after a specified position.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var nextCodePointIndex = require( '@stdlib/string/next-code-point-index' );
+```
+
+#### nextCodePointIndex( string\[, fromIndex] )
+
+Returns the position of the next Unicode code point in a string after a specified position.
+
+```javascript
+var out = nextCodePointIndex( 'last man standing' );
+// returns 1
+```
+
+By default, the function searches for a Unicode code point starting from the first index. To specify an alternative starting search index, provide a `fromIndex` argument.
+
+```javascript
+var out = nextCodePointIndex( 'last man standing', 4 );
+// returns 5
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   If `string` is an empty string, the function returns `-1` irrespective of `fromIndex`.
+-   If an code point does not exist after `fromIndex`, the function returns `-1`.
+-   Note that `fromIndex` does **not** refer to a visual character position, but to an index in the ordered sequence of [UTF-16][utf-16] code units.
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var nextCodePointIndex = require( '@stdlib/string/next-code-point-index' );
+
+var out = nextCodePointIndex( 'last man standing', 4 );
+// returns 5
+
+out = nextCodePointIndex( 'presidential election', 8 );
+// returns 9
+
+out = nextCodePointIndex( 'अनुच्छेद', 1 );
+// returns 2
+
+out = nextCodePointIndex( '🌷', 0 );
+// returns -1
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section for describing a command-line interface. -->
+
+* * *
+
+<section class="cli">
+
+## CLI
+
+<!-- CLI usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```text
+Usage: next-code-point-index [options] [<string>]
+
+Options:
+
+  -h,    --help                Print this message.
+  -V,    --version             Print the package version.
+         --from index          Starting search position in string. Default: 0.
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- CLI usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- CLI usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```bash
+$ next-code-point-index --from=1 अनुच्छेद
+3
```

Comment:
This return result does not agree with https://github.com/stdlib-js/stdlib/pull/1117/files#diff-0cd08fc9fda2c8623ebe39c0bbfbbb3facca313787f7a84b5838afd9a19ca240R95. Should this be changed?